src/Eccube/Resource/template/admin/Setting/Shop/mail.twig

```diff
@@ -62,58 +62,77 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 {% endblock javascript %}
 
 {% block main %}
-
-    <div class="row" id="aside_wrap">
-        <form name="form1" role="form" class="form-horizontal" id="form1" method="post" action="">
-            {{ form_widget(form._token) }}
-
-            <div id="mail_edit_box" class="col-md-9">
-                <div id="mail_edit_box__body" class="box">
-                    <div id="mail_edit_box__header" class="box-header">
-                        <h3 class="box-title">{{'admin.setting.shop.mail.593'|trans}}</h3>
-                    </div><!-- /.box-header -->
-                    <div id="mail_edit_box__body_inner" class="box-body">
-
-                        {{ form_row(form.template) }}
-                        {{ form_row(form.mail_subject) }}
-                        <div id="detail_box__tpl_data" class="form-group">
-                            {{ form_label(form.tpl_data) }}
-                            <div class="col-sm-9 col-lg-10">
-                                <div id="editor" style="height: 480px"></div>
-                                {{ form_widget(form.tpl_data) }}
-                                {{ form_errors(form.tpl_data) }}
+<form name="form1" role="form" class="form-horizontal" id="form1" method="post" action="">
+    {{ form_widget(form._token) }}
+    <div class="c-contentsArea__cols">
+        <div class="c-contentsArea__primaryCol">
+            <div class="c-primaryCol">
+                <div class="card rounded border-0 mb-4">
+                    <div class="card-header"><span class="card-title">{{ 'admin.setting.shop.mail.mail_template_edit'|trans }}</span></div>
+                    <div class="card-body">
+                        <div class="row mb-2">
+                            <div class="col-2"><span>{{ 'admin.setting.shop.mail.template'|trans }}</span><i class="fa fa-question-circle fa-lg ml-1"></i></div>
+                            <div class="col-10">
+                                {{ form_widget(form.template) }}
```

Comment:
@hoand-vn 
When we change from `form_row` to` form_widget`, we also need to add `form_errors`.

```
{{ form_row(form.template) }}
```
↓
```
{{ form_widget(form.template) }}
{{ form_errors(form.template) }}
```
There are four places in this file, so please fix them.

Reply:
@okazy 
fixed your comment. please check it

Reply:
@hoand-vn 
OK, Thanks you!